homeassistant/const.py

```diff
@@ -180,6 +180,7 @@
 CONF_ZONE = "zone"
 
 # #### EVENTS ####
+EVENT_AUTOMATION_RELOADED = "automation_reloaded"
```

Comment:
Why is one in `const.py` and the other not? I think that it would be fine to both be in their integrations as they are integration specific.

Reply:
I tried to stay consistent with where `EVENT_AUTOMATION_TRIGGERED` is located.

Quick search, I guess that one is mainly in that location for the logbook.

Reply:
yes let's do that. 

And automation should really just register a `subscribe_event` with logbook ;)

Reply:
Yes, it should 😉

Reply:
done: #36356